scripts/ASTImportTest.sh

```diff
@@ -1,10 +1,41 @@
 #!/usr/bin/env bash
+# ------------------------------------------------------------------------------
+# vim:ts=4:et
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) solidity contributors.
+# ------------------------------------------------------------------------------
+
+set -euo pipefail
+
+REDIRECT_TO="/dev/null"
+IMPORT_TEST_TYPE=
+for PARAM in "$@"
+do
+  case "${PARAM}" in
```

Comment:
I guess we can apply here the same rule that we are currently following in solc-js scripts and only use `"${a}"` form when the variable is between a string and `"$a"` when it is not.

```suggestion
  case "$PARAM" in
```

Reply:
I find it cooler cooler to always write `${..}` :D - but yeah, I will change that

Reply:
haha I agree, I used to use it always. But @cameel made a good point here https://github.com/ethereum/solc-js/pull/659#discussion_r975511794 about only use it where it is needed, and I started to monitor myself since then, haha